village/src/App.js

```diff
@@ -11,14 +19,48 @@ class App extends Component {
       smurfs: [],
     };
   }
+  componentDidMount(){
+    axios
+    .get('http://localhost:3333/smurfs')
+    .then(response => {
+      this.setState({smurfs: response.data});
+      console.log(this.state.smurfs)
+    })
+    .catch(error => {
+      console.error('Server Error', error);
+      });
+  }
+
+  addNewSmurf = (obj) => {
```

Comment:
nice methods could you give me an explanation of your workflow and thoughts when structuring this app